### PR TITLE
Update copy on /tco-calculator

### LINKED ISSUE #7579: [COPY UPDATE] tco-calculator - "Private cloud TCO calculator"
Please update tco-calculator - "Private cloud TCO calculator" from the [copy doc](https://docs.google.com/document/d/1n3ExZwFVlw5LyAp7O3CgPthVgx8yRX5-Wg_gBQLf73c/edit?usp=drivesdk)

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Update copy on /tco-calculator with a link in the banner and a button for contact after the caclulator

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/tco-calculator
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to and resolve comments in the [copy doc](https://docs.google.com/document/d/1n3ExZwFVlw5LyAp7O3CgPthVgx8yRX5-Wg_gBQLf73c/edit#)

## Issue / Card

Fixes #7579

### CODE REVIEW

File: templates/tco-calculator/index.html [hunk 1]
```diff
@@ -10,8 +10,12 @@
 <section class="p-strip--suru is-dark">
   <div class="row">
     <div class="col-8">
-      <h1>Private cloud TCO calculator</h1>
-      <p>One of the biggest promises behind the private cloud infrastructure is better economics compared to proprietary hardware and public clouds. Use the TCO (total cost of ownership) calculator below to estimate the cost of your private cloud infrastructure with Canonical.</p>
+      <h1>
+        Private cloud TCO calculator
+      </h1>
+      <p>
+        One of the biggest promises behind the private cloud infrastructure is better economics compared to proprietary hardware and public clouds. Use the TCO (total cost of ownership) calculator below to estimate the cost of your private cloud infrastructure with Canonical or <a class="p-link--inverted" href="/openstack/compare">learn more about differences between various OpenStack distributions</a>.
```
Comment (SirSamTumless) [inline templates/tco-calculator/index.html:17]: You can't see that this is a link, as it is the same colour as the rest of the text.

File: templates/tco-calculator/index.html [hunk 2]
```diff
@@ -221,7 +225,12 @@ <h2 class="p-heading--three">Self-managed</h2>
   </div>
 
   <div class="u-fixed-width">
-    <p class="p-heading--six"><em>NOTE: This calculator is for informational purposes only. Actual numbers may vary depending on your requirements. Also, discounts may apply for bigger clusters. For more detailed pricing, <a href="/openstack/managed#get-in-touch">contact Canonical’s sales team</a>.</em></p>
+    <p class="p-heading--six"><em>NOTE: This calculator is for informational purposes only. Actual numbers may vary depending on your requirements. Also, discounts may apply for bigger clusters. For more detailed pricing, please contact Canonical’s sales team.</em></p>
+    <p>
+      <a class="p-button--positive js-invoke-modal" href="/openstack/managed#get-in-touch">
```
Comment (SirSamTumless) [inline templates/tco-calculator/index.html:230]: This link doesn't work for me.

File: templates/tco-calculator/index.html (hunk 1 above)
Comment (pmahnke) [reply templates/tco-calculator/index.html:17]: That's a Vanilla thing? sorry

File: templates/tco-calculator/index.html (hunk 2 above)
Comment (pmahnke) [reply templates/tco-calculator/index.html]: Fixed

File: templates/tco-calculator/index.html (hunk 1 above)
Comment (SirSamTumless) [reply templates/tco-calculator/index.html:17]: Sweet!

**Pull request merged**